src/ChainPriorities.hpp

```diff
@@ -4,9 +4,12 @@
 #define TH_CHAIN_PRIO_CALC_ASCIIMANAGER 1
 #define TH_CHAIN_PRIO_CALC_MAINMENU 2
 #define TH_CHAIN_PRIO_CALC_GAMEMANAGER 4
+#define TH_CHAIN_PRIO_CALC_STAGE 6
 
 #define TH_CHAIN_PRIO_DRAW_MAINMENU 0
 #define TH_CHAIN_PRIO_DRAW_GAMEMANAGER 2
+#define TH_CHAIN_PRIO_DRAW_HIGH_PRIO_STAGE 3
```

Comment:
Not sure about this name tho, idk if it should be like that or `TH_CHAIN_HIGH_PRIO_DRAW_STAGE`

Reply:
I'd rather not alter the TH_CHAIN_PRIO prefix. This is fine as-is :)